src/effects/effect_1.c

```diff
@@ -1,16 +1,185 @@
 #include "common.h"
+#include "effects_internal.h"
 
-INCLUDE_ASM(s32, "effects/effect_1", fx_1_main);
+typedef struct Effect1 {
+    /* 0x00 */ s16 unk_00;
+    /* 0x02 */ s16 unk_02;
+    /* 0x04 */ s16 unk_04;
+    /* 0x06 */ s16 unk_06;
+    /* 0x08 */ s32 unk_08;
+    /* 0x0C */ f32 x;
+    /* 0x10 */ f32 y;
+    /* 0x14 */ f32 z;
+    /* 0x18 */ f32 partX;
+    /* 0x1C */ f32 partY;
+    /* 0x20 */ f32 unk_20;
+    /* 0x24 */ f32 unk_24;
+    /* 0x28 */ char unk_28[0x4];
+} Effect1; // size = 0x2C
 
-void func_E00021B4(void) {
+void fx_1_init(EffectInstance* effect);
+void fx_1_update(EffectInstance* effect);
+void fx_1_render(EffectInstance* effect);
+void fx_1_appendGfx(EffectInstance* effect);
+
+f32 D_E0002760[10] = { 10.0f, 40.0f, 80.0f, 170.0f, 140.0f, 100.0f, 25.0f, 155.0f, 60.0f, 120.0f };
+f32 D_E0002788[10] = { 2.2f, 2.7f, 3.0f, 2.2f, 2.7f, 3.0f, 1.9f, 1.9f, 1.5f, 1.5f };
+f32 fx_1_partScales[10] = { 1.4f, 1.3f, 1.2f, 1.3f, 1.4f, 1.3f, 1.6f, 1.6f, 1.6f, 1.6f };
+f32 fx_1_partYaws[10] = { 0.0f, 234.0f, 468.0f, 702.0f, 936.0f, 1260.0f, 1404.0f, 1638.0f, 1902.0f, 1976.0f };
+
+s32 fx_1_dlists[7] = { 0x09000FA0, 0x09001060, 0x09001120, 0x090011E0, 0x090012A0, 0x09001360, 0x09001420 };
```

Comment:
I think it would make sense to make these static, then we can just call them `partScales` etc

Reply:
Yeah but then their scope is unclear. should I prefix them with s ?

Reply:
IDM on prefix, I personally don't think any is needed but its your call